lib/pure/math.nim

```diff
@@ -56,7 +56,7 @@ type
     fcNegInf     ## value is negative infinity
 
 proc classify*(x: float): FloatClass =
-  ## classifies a floating point value. Returns `x`'s class as specified by
+  ## Classifies a floating point value. Returns `x`'s class as specified by
```

Comment:
So after discussion what is conclusion about proper format? Big letter or small letter? I need to know for my codes and their comment because I want my codes and their comment to follow Nim convention at all time!